### DISCUSSION TITLE
CI/CD Pipeline

### DISCUSSION THREAD

**Discussion by kartikayasijaa** [kartikayasijaa/talk-trove#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kartikayasijaa** [comment]
Hey, anyone wants to work on this issue #10 with me? We'll learn alot of things by implementing this.

**Reply by ChampGupta** [reply]
Hey, I would like to contribute on this issue with you

**Comment by Sibam-Paul** [comment]
@kartikayasijaa @ChampGupta will like to assist u
